dbsync-receiver-app/pom.xml

```diff
@@ -33,7 +33,7 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
-                    <attach>false</attach>
+                    <attach>true</attach>
```

Comment:
attach needs to be set to false otherwise the jar pushed to the maven repo can't be used a dependency